styles.css

```diff
@@ -610,3 +660,491 @@ li {
 #button_contact:active {
   background-color: #008552;
 }
+
+@media all and (min-width: 768px) {
+  .header2 {
+    display: none;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  body {
+    height: 3846px;
+    background-color: #fff;
+    background-image: url(/img/imgdes/img_form1.png), url(/img/imgdes/img_fom2.png), url(/img/imgdes/img_form3.png);
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-size: 250px 360px, 130px 106px, 223px 495px;
+    background-position: 0 3650px, 280px 3800px, 1220px 3510px;
+  }
+
+  #heading {
+    background-image: url(/img/imgdes/background_header.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 1472.8px;
```

Comment:
- [ ] I think because of the static and exact widths like this in the selector and other selectors, your design has responsiveness issues that you need to solve as mentioned by the previous reviewer as well. the width on some of elements are beyond `1440px` and this created a horizontal scroll on your design. Generally to ensure responsive design, we avoid giving exact specific widths to our design. kindly take a look at this [link](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-introduction-to-responsive-design) for more details.
- I think because of the static width size, it is not aligning all of your content in the center. Please kindly make sure that the content of your entire application is positioned in the center. Following sections should be positioned in the middle of the screen:

Introduction
Recent Works
About Me
Contact
Kindly consider fixing this, please. 👍🏽

Reply:
@Zuhaib042 I think another problem is that I use position properties to organize the elements, I know I made a mistake in the first project, and I'm a little sad about that but I have to improve the positioning of the elements, I think the best thing would be to repeat the module to organize the mobile project well and then move on, I thought I was doing it well but with this project, I understood the importance of organizing the elements correctly.